### ISSUE TITLE
<menuitem> will be disabled in Firefox v85

### ISSUE THREAD

**Issue by tgn-ff** [greasemonkey/greasemonkey#3099]
See [Bug 1680596](https://bugzilla.mozilla.org/show_bug.cgi?id=1680596). I noticed it on Nightly due to the disabled scripts not being faded anymore.

**Comment by StigNygaard** [comment]
See #3078

**Comment by arantius** [comment]
I see it still working in 85.0b9 .

**Comment by tgn-ff** [comment]
`dom.menuitem.enabled` defaults to false in 85.0b9 on my end.

**Comment by arantius** [comment]
Did a fresh install of Nightly and
![image](https://user-images.githubusercontent.com/84729/104925750-39258680-596d-11eb-9fd0-c0c5fcf0e9d4.png)
I see the menu command polyfill still work fine, in 86.0a1.

The "disabled scripts not greyed out" issue, I do see.  I'm confused about whether that's relevant to this change or not.  Rivets/tinybind is just not setting the "disabled" attribute, and this persists even if I change \<menuitem\>s to \<div\>s.

**Comment by StigNygaard** [comment]
@arantius I don't understand your screenshot. What are you trying to show? the "Take a Screenshot" item is not created from html5 <menuitem>, it's a standard feature of Firefox. Isn't it?
Also the html5 menuitem is/was placed in top of context-menu. You can use webextension API though to add items in bottom of contextmenu.

Or am I totally misunderstanding what you are trying to show?

**Comment by arantius** [comment]
I was confused.  (The GM.rmc command I use most often is from the "Search with Google" script, because Google is not the default engine in my main profile.  In this test profile I installed it and _thought_ I saw it run and create a menu item.  Nope, that's the standard (Google is the default by default!) search context item.)

Install https://gist.github.com/arantius/10f7a8c998c2d06d84b8f9dad48b3f92 in Firefox stable (84) and you see a context item:

![image](https://user-images.githubusercontent.com/84729/104957333-f5964100-599b-11eb-8b15-ebc3550b6ec1.png)

Install it in Nightly (86), nothing.  Report confirmed.
